apps/server/src/modules/room/api/test/room-remove-members.api.spec.ts

```diff
@@ -249,6 +249,19 @@ describe('Room Controller (API)', () => {
 
 				expect(response.status).toBe(HttpStatus.FORBIDDEN);
 			});
+
+			describe('when admin is member in the room', () => {
```

Comment:
reminder: only when not room owner I guess

Reply:
added tests for double role:
- one where user is schooladmin + room owner => FORBIDDEN
- one where user is schooladmin + room viewer => OK